vite.config.ts

```diff
@@ -1,8 +1,18 @@
 import react from '@vitejs/plugin-react';
-import { defineConfig } from 'vite';
 import loadVersion from 'vite-plugin-package-version';
+import { defineConfig } from 'vitest/config';
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react(), loadVersion()],
+  test: {
+    server: {
+      deps: {
+        // this package does not export esm that uses imports with the file extension, this is a workaround
+        // to instruct vitest to bundle the package similar to the app
+        // https://stackoverflow.com/questions/77126321/how-to-get-vitest-working-with-require-of-es-module
+        inline: ['@ugrc/utah-design-system'],
+      },
+    },
```

Comment:
**suggestion**: i just released the packages with the .js extension if you want to try updating and removing this.

Reply:
The new versions appears to have fixed this issue.